src/core/js/helpers.js

```diff
@@ -355,6 +355,18 @@ define([
         a11y_attrs_tabbable: function() {
             Adapt.a11y.log.deprecated('a11y_attrs_tabbable should not be used. tabbable elements should be natively tabbable.');
             return new Handlebars.SafeString(' role="region" tabindex="0" ');
+        },
+
+        /**
+         * Produce display text with alternative screen reader version.
+         * @param {string} visible Visual text
+         * @param {string} alternatives Any number of additional arguments which will be concatenated and read
```

Comment:
Text that will be read by the screen reader (instead of what's displayed on screen)